src/ansys/materials/manager/util/fluent/writer_fluent.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2022 - 2025 ANSYS, Inc. and/or its affiliates.
```

Comment:
This module could just be called `writer`, the fact that it's in the `fluent` module tells you it's for fluent anyway.

It could then be imported as `from .util.fluent import writer as FluentWriter`.